src/web/src/components/SignUp/Forms/RSSFeeds.tsx

```diff
@@ -0,0 +1,263 @@
+      position: 'absolute',
+      minHeight: '100%',
+      width: '100%',
+      [theme.breakpoints.down(600)]: {
+        width: '95%',
+        marginLeft: '2.5%',
+      },
+    },
+    blogPageTitle: {
+      fontSize: '1.5em',
+    },
+    helpText: {
+      fontSize: '1.1em',
+      lineHeight: '1.8em',
+    },
+    infoContainer: {
+      display: 'grid',
+      textAlign: 'center',
+      gridGap: '10%',
+      justifyItems: 'center',
+      alignItems: 'center',
+      width: '90%',
+    },
+    inputsContainer: {
+      width: '100%',
+      display: 'grid',
+      gridTemplateColumns: '75% 25%',
+      '& .MuiFormHelperText-root': {
+        fontSize: '0.9em',
+        color: 'black',
+      },
+      '& .MuiFormLabel-root': {
+        color: 'black',
+      },
+      [theme.breakpoints.down(600)]: {
+        gridTemplateColumns: '80% 20%',
+      },
+    },
+    helpMessage: {
+      fontSize: '.9em',
+      color: 'black',
+    },
+    button: {
+      height: '35px',
+      width: '50%',
+      alignSelf: 'center',
+      fontSize: '0.8em',
+      marginLeft: '5%',
+      background: '#121D59',
+      color: '#A0D1FB',
+      '&:hover': {
+        color: 'black',
+        border: '1px solid #121D59',
+      },
+      '&.Mui-disabled': {
+        backgroundColor: 'inherit',
+      },
+    },
+    RssButtonContainer: {
+      width: '90%',
+      display: 'grid',
+    },
+    infoRSSContainer: {
+      minHeight: '120px',
+      maxHeight: '120px',
+      width: '100%',
+      overflowY: 'auto',
+    },
+    noBlogMessage: {
+      fontSize: '1em',
+      color: '#474747',
+      marginTop: '40px',
+    },
+    text: {
+      fontSize: '0.9em',
+      alignSelf: 'end',
+      color: '#474747',
+    },
+    RssButtonWrapper: {
+      width: '100%',
+    },
+    RssButton: {
+      width: '101%',
+      borderRadius: '0',
+      background: '#121D59',
+      color: '#A0D1FB',
+      '&:hover': {
+        color: 'black',
+        border: '1px solid #121D59',
+      },
+    },
+    agreeMessage: {
+      [theme.breakpoints.down(600)]: {
+        alignSelf: 'end',
+      },
+    },
+    formControlLabel: {
+      fontSize: '.9em',
+      height: '10px',
+      color: '#474747',
+    },
+  })
+);
+
+const RSSFeeds = connect<{}, SignUpForm>((props) => {
+  const classes = useStyles();
+  const { values, errors, setFieldValue } = props.formik;
+  const { token } = useAuth();
+
+  const [feedUrls, setFeedUrls] = useState<Array<string>>([]);
+  const [blogUrlError, setBlogUrlError] = useState('');
+  const [validating, setValidating] = useState(false);
+  const controllerRef = useRef<AbortController | null>();
+
+  const validateBlog = async () => {
+    if (!errors.blogUrl) {
+      try {
+        setValidating(true);
+        if (controllerRef.current) {
+          controllerRef.current.abort();
+        }
+        controllerRef.current = new AbortController();
+        const response = await fetch(`${feedDiscoveryServiceUrl}`, {
```

Comment:
`fetch(feedDiscoveryServiceUrl)` with out the `${}` since it's already a string.